apps/gp2-server/src/routes/user.route.ts

```diff
@@ -56,7 +56,9 @@ export const userRouteFactory = (userController: UserController): Router => {
   userRoutes.get<{ userId: string }>('/users/:userId', async (req, res) => {
     const { userId } = validateUserParameters(req.params);
 
-    const user = await userController.fetchById(userId);
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const loggedInUserId = req.loggedInUser!.id;
+    const user = await userController.fetchById(userId, loggedInUserId);
```

Comment:
dont we know here if it it the same id? and we could just pass a boolean?

Reply:
Yes, I was looking to put all the logic in the controller and leave the routing to do routes. Doing it like this is consistent with the project and working groups controllers - where they too take the loggedInUserId